uiowa/uiowa
# Feature: Follow-up additions to webform block options and form label a11y

This resolves https://github.com/uiowa/uiowa/issues/2531. 

# In this PR: 

- Adds a white background color to webform block options that didn't make it in https://github.com/uiowa/uiowa/pull/2502
- Removes specific form label color overrides from `form.scss` in favor of the inherit rules set in `background.scss`
- Adds `templates > form > container.html.twig` in order to remove duplicate background color classes being added when editing a layout builder page and causing color contrast issues noted in https://github.com/uiowa/uiowa/issues/2531

## How to test
- Test https://sandbox.prod.drupal.uiowa.edu/kitchen-sink and edit the page and add a white background color to one of the contact forms and it should add a white background color. 
- Test https://sandbox.prod.drupal.uiowa.edu/kitchen-sink and edit the page in layout builder and verify that you can see the form labels and buttons within the text area like black on black or gold on gold. 
- Test page of https://sandbox.prod.drupal.uiowa.edu/kitchen-sink and verify that the contact form labels don't have any color contrast issues like black on black brain pattern.
